CCSMB-1: Procedure for Standards Submission (#1)
CCSMB/Standards: Standards/CCSMB-1.md

> # CCSMB-1: Procedures for submitting a standard

Reviewer: This should be capitalized according to standard English APA/Whatever rules
https://capitalizemytitle.com/#capitalizationrules

Author: Maybe reword it a bit too:
```suggestion
# CCSMB-1: Procedures for Standards Submission
```

This RFC defines the protocol for defining and submitting a standard to the CCSMB.

> ## File names and formatting

Reviewer: This should *maybe* be capitalized according to standard rules.

Author: ```suggestion
## File Naming and Formatting
```

Each RFC should be named `CCSMB-n.md`, where `n` is a number counting upwards.

> Each RFC should be a [GitHub-Flavored Markdown](https://github.github.com/gfm/) document with American English spelling.

Reviewer: The way it has always been in CC is British English spelling.

Author: Ah, let's do that then.
```suggestion
Each RFC should be a [GitHub-Flavoured Markdown](https://github.github.com/gfm/) document with British English spelling.
```

> Each RFC should be a [GitHub-Flavored Markdown](https://github.github.com/gfm/) document with American English spelling.

Reviewer: The way it has always been in CC is British English spelling.

Author: Resolved in [`75376de`](https://github.com/CCSMB/Standards/pull/1/commits/75376de185bc0b3565e8a6c7fadc956ff14469a0)
